charts/sourcegraph/values.yaml

```diff
@@ -104,67 +130,97 @@ cadvisor:
       cpu: 150m
       memory: 200Mi
   serviceAccount:
+    # -- Enable creation of ServiceAccount for `cadvisor`
     create: true
+    # -- Name of the ServiceAccount
     name: cadvisor
 
 codeInsightsDB:
+  # -- Enable `codeinsights-db` PostgreSQL server
   enabled: true
+  # -- Environment variables for the `codeinsights-db` container
   env:
     POSTGRES_PASSWORD: # Accessible by Sourcegraph applications on the network only, so password auth is not used.
       value: password
+  # -- Name of existing ConfigMap for `codeinsights-db`. It must contain a `postgresql.conf` key.
   existingConfig: "" # Name of an existing configmap
   image:
+    # -- Docker image tag for the `codeinsights-db` image
     defaultTag: 3.37.0@sha256:fa608333a6ca1aef148abd33e4ee14886d4f172e0db1e5c9ee6bac36adec0bf1
+    # -- Docker image name for the `codeinsights-db` image
     name: "codeinsights-db"
+  # -- Security context for the `codeinsights-db` container,
+  # learn more from the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container)
   containerSecurityContext:
     allowPrivilegeEscalation: false
     runAsUser: 70
     runAsGroup: 70
     readOnlyRootFilesystem: true
+  # -- Number of `codeinsights-db` pod (this should always be `1`)
   replicaCount: 1
```

Comment:
Maybe we should remove this setting as an option for the databases and hardcode the count into the template?

Reply:
> Maybe we should remove this setting as an option for the databases and hardcode the count into the template?

I would love that